lib/util/http/types.ts

```diff
@@ -70,6 +70,7 @@ export interface HttpOptions {
 
 export interface HttpHeaders extends IncomingHttpHeaders {
   link?: string | undefined;
+  'x-ausername'?: string; // bitbucket-server
```

Comment:
i don't like this addition. do we really need this? can't we use `isNotEmptyAndNotWhitespaceString` instead?